src/cpp/elf/aie2/aie2_blob_elfwriter.h

```diff
@@ -34,8 +34,12 @@ class aie2_config_elf_writer: public elf_writer
    * It add partition info in note section and finalize to generate elf
    */
   std::vector<char>
-  process(std::vector<std::shared_ptr<writer>>& mwriter) override
+  process(std::vector<std::shared_ptr<writer>> &mwriter) override
```

Comment:
Uh oh, I don't like this, it's uncommon in C++to have space before `*` and `&`, I favor Stroustrup's style.